yaksh/python_assertion_evaluator.py

```diff
@@ -43,6 +39,7 @@ def compile_code(self):
             submitted = compile(self.user_answer, '<string>', mode='exec')
             self.exec_scope = {}
             exec(submitted, self.exec_scope)
+            exec("from nose.tools import *", self.exec_scope)
```

Comment:
Please don't use star, since we do not know what this scope will include given that student code could be anything.

Reply:
Nose is added to the scope for the moderator so that he can use nose asserts on student's code